src/utils/templates/apply-functions/apply-sourceurl-template.ts

```diff
@@ -0,0 +1,8 @@
+import { NoteData } from '../../../models/NoteData';
+import * as P from './../placeholders/sourceurl-placeholders';
+import { applyConditionalTemplate } from './apply-conditional-template';
+
+export const applySourceUrlTemplate = (noteData: NoteData, text: string): string => {
+    console.log("XXX",noteData.sourceUrl);
```

Comment:
This was probably only meant for debugging.

Reply:
Yes indeed 😅 . I removed it!